<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dc_ioClass</class>
 <widget class="QMainWindow" name="dc_ioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dc_io</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QWidget" name="Display" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="PlayButton">
      <property name="text">
       <string>Play</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="CaptureButton">
      <property name="text">
       <string>Capture</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="RecordButton">
      <property name="text">
       <string>Record</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionRaw"/>
    <addaction name="actionRegistered"/>
    <addaction name="action3D"/>
   </widget>
   <widget class="QMenu" name="menu_Set">
    <property name="title">
     <string>&amp;Set</string>
    </property>
    <widget class="QMenu" name="menuScale">
     <property name="title">
      <string>&amp;Scale</string>
     </property>
     <addaction name="action2"/>
     <addaction name="action1"/>
     <addaction name="action05"/>
    </widget>
    <addaction name="actionROI"/>
    <addaction name="actionROD"/>
    <addaction name="separator"/>
    <addaction name="menuScale"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menu_Set"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRaw">
   <property name="text">
    <string>&amp;Raw</string>
   </property>
  </action>
  <action name="actionRegistered">
   <property name="text">
    <string>R&amp;egistered</string>
   </property>
  </action>
  <action name="action3D">
   <property name="text">
    <string>3&amp;D</string>
   </property>
  </action>
  <action name="actionROI">
   <property name="text">
    <string>RO&amp;I</string>
   </property>
  </action>
  <action name="actionROD">
   <property name="text">
    <string>RO&amp;D</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action05">
   <property name="text">
    <string>0.5</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dc_io.qrc"/>
 </resources>
 <connections/>
</ui>
